mozilla-vpn/explores/funnel.explore.lkml

```diff
@@ -0,0 +1,56 @@
+include: "../views/*.view"
+
+explore: funnel {
+  view_name: login_flows
```

Comment:
Consider renaming all of these to event-based names (e.g. begin_funnel, graduated_to_user, graduated_to_strip_customer, etc.)